frontend/app/terms/page.tsx

```diff
@@ -0,0 +1,263 @@
+        </Heading>
+        <Heading as="h2" size="md" mt={10}>
+          1. Asset Revival Fees
+        </Heading>
+        <Text mt={4}>
+          The revival of assets from the GRAVE back to your Universal Profile is
+          subject to a fee. The specific fee structure is as follows:
+        </Text>
+        <List spacing={2} mt={4}>
+          <ListItem>
+            A fixed fee of 0.005 LYX is charged per revival action.
+          </ListItem>
+          <ListItem>
+            Fees are subject to change at our discretion. We will endeavor to
+            notify you of any fee changes in a timely manner.
+          </ListItem>
+          <ListItem>
+            Fees are non-refundable except as required by law or as explicitly
+            set forth in these Terms.
+          </ListItem>
+        </List>
+        <Text mt={4}>
+          Please ensure that you are aware of and agree to the applicable fees
+          before using the service to revive assets from the GRAVE.
+        </Text>
+
+        {/* Risk Disclaimer */}
+        <Heading id="disclaimer" as="h1" size="lg" mb={4} mt={20}>
+          Risk Disclaimer
+        </Heading>
+        <Heading as="h2" size="md" mt={10}>
+          1. General Risks
+        </Heading>
+        <Text mt={4}>
+          You acknowledge that interacting with blockchain technology and
+          digital assets involves significant risks including, but not limited
+          to, the risk of financial loss, the volatility of digital assets, and
+          the risk of unforeseen legal implications. You agree to assume all
+          such risks associated with the use of Universal Grave.
+        </Text>
+        <Heading as="h2" size="md" mt={10}>
+          2. No Warranty
+        </Heading>
+        <Text mt={4}>
+          Universal Grave is provided "as is" and "as available" without any
+          warranties, express or implied. We do not guarantee the continuous,
+          uninterrupted, or error-free operability of the services.
+        </Text>
+
+        {/* Privacy Policy */}
+        <Heading id="privacy" as="h1" size="lg" mb={4} mt={20}>
+          Privacy Policy
+        </Heading>
+        <Heading as="h2" size="md" mt={10}>
+          1. Data Collection and Use
+        </Heading>
+        <Text mt={4}>
+          While Universal Grave operates on blockchain technology and does not
+          directly collect personal data, please be aware that all transactions
+          are public and immutable due to the nature of blockchain.
+        </Text>
+
+        {/* Intellectual Property Rights */}
+        <Heading id="ip" as="h1" size="lg" mb={4} mt={20}>
+          Intellectual Property Rights
+        </Heading>
+        <Heading as="h2" size="md" mt={10}>
+          1. Ownership
+        </Heading>
+        <Text mt={4}>
+          Universal Grave, its original content, features, and functionality are
+          and will remain the exclusive property of [Your Company Name] and its
```

Comment:
[Your Company Name]

I feel that is on purpose for the future, but just checking

Reply:
yes ser. there's another ticket handling this for the future